db/migrate/20231018113402_change_objective_fields_to_dates.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ChangeObjectiveFieldsToDates < ActiveRecord::Migration[6.1]
```

Comment:
It's ok to rollback and rework migrations while it's in dev. No need to create new one.